package.json

```diff
@@ -151,6 +151,7 @@
     "@feathersjs/feathers": "5.0.5",
     "@feathersjs/schema": "5.0.5",
     "@feathersjs/typebox": "5.0.5",
+    "@hookstate/subscribable": "^4.0.0",
```

Comment:
incorrect merge, we have removed this dep

Reply:
removed